src/System.Security.Cryptography.Csp/tests/Configurations.props

```diff
@@ -4,6 +4,8 @@
     <BuildConfigurations>
       netstandard-Windows_NT;
       netcoreapp-Windows_NT;
+      netstandard-Unix;
```

Comment:
Could you please run from the root of the repo `msbuild /t:UpdateVSConfigurations build.proj` This will change some configurations on the test csproj so that we don't break VS with this change.